pkg/manager/member/utils.go

```diff
@@ -269,7 +271,7 @@ func MapContainers(podSpec *corev1.PodSpec) map[string]corev1.Container {
 }
 
 // updateStatefulSet is a template function to update the statefulset of components
-func updateStatefulSet(setCtl controller.StatefulSetControlInterface, tc *v1alpha1.TidbCluster, newSet, oldSet *apps.StatefulSet) error {
```

Comment:
Please revert this, no need to change this in release-1.1.

Reply:
reverted